### PR TITLE
Set better default for new Firefox on Android

### PR DESCRIPTION
**Pull request by soulgalore**
#1331

### CODE REVIEW

File: lib/firefox/webdriver/builder.js [hunk 1]
```diff
@@ -218,11 +218,20 @@ module.exports.configureBuilder = function(builder, baseDir, options) {
       }
     } else {
       ffOptions.firefoxOptions_().androidPackage = 'org.mozilla.firefox';
```
Comment (gmierz) [inline lib/firefox/webdriver/builder.js:220]: Are you leaving it as this (rather than org.mozilla.firefox_beta) in preparation for the release version?

**Review by gmierz — APPROVED** [review]
Looks good thanks! Just a small question about the package name.

File: lib/firefox/webdriver/builder.js (hunk 1 above)
Comment (soulgalore) [reply lib/firefox/webdriver/builder.js:220]: Yep, thinking there is only a couple of days more to wait?

Comment (gmierz) [reply lib/firefox/webdriver/builder.js:220]: Sounds good. The last I heard was that it should make it into a release on around ~Sept 22nd which is about a week away.

**Pull request merged**